sql/npc_list.sql

```diff
@@ -1346,49 +1346,49 @@ INSERT INTO `npc_list` VALUES (16859197,'2','     ',0,-80.000,-1.000,-40.000,7,4
+INSERT INTO `npc_list` VALUES (16859459,'_0k7','warp_09',0,360.001,-0.399,-40.001,1,40,40,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859460,'_0k8','warp_10',0,-320.002,-0.400,-359.995,1,40,40,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859461,'_0k9','warp_11',0,-39.997,-0.400,-319.997,1,40,40,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859462,'_0ka','Memory Flux',0,80.000,-0.400,-239.999,1,40,40,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859463,'_0kb','Memory Flux',0,240.003,-0.400,-319.994,1,40,40,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16859464,'_0k0','Memory Flux',0,260.000,-2.500,180.000,1,40,40,9,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16859465,'_0kc','Memory Flux',0,300.000,-2.500,-220.000,1,40,40,9,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16859466,'_0kd','Memory Flux',0,-260.000,-2.500,-180.000,1,40,40,9,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16859467,'kiokudama_a','kiokudama_a',0,299.920,-0.044,179.982,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'COP',1);
-INSERT INTO `npc_list` VALUES (16859468,'kiokudama_b','kiokudama_b',0,340.261,-0.053,-220.029,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'COP',1);
-INSERT INTO `npc_list` VALUES (16859469,'kiokudama_c','kiokudama_c',0,-299.906,-0.045,-180.055,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'COP',1);
-INSERT INTO `npc_list` VALUES (16859470,'qm1','???',0,280.001,-2.328,280.000,32774,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859465,'_0kc','kiokudama_a',0,300.000,-2.500,-220.000,1,40,40,9,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859466,'_0kd','kiokudama_b',0,-260.000,-2.500,-180.000,1,40,40,9,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16859467,'kiokudama_a','kiokudama_c',0,299.920,-0.044,179.982,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'COP',1);
+INSERT INTO `npc_list` VALUES (16859468,'kiokudama_b','???',0,340.261,-0.053,-220.029,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'COP',1);
+INSERT INTO `npc_list` VALUES (16859469,'kiokudama_c','',0,-299.906,-0.045,-180.055,32769,40,40,0,0,0,0,2051,0x0000340000000000000000000000000000000000,0,'COP',1);
+INSERT INTO `npc_list` VALUES (16859470,'qm1','Moogle',0,280.001,-2.328,280.000,32774,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'COP',0);
```

Comment:
this is not sync'd up properly

some manual review is required even with the auto tools